docs/topics/jvm/jvm-spring-boot-restful-db.md

```diff
@@ -0,0 +1,186 @@
+
+## Configure the database
+
+Configure the database in the application:
+
+1. Create a new folder called `sql` in the `src/main/resources` with the `scheme.sql` file inside. It will store the database scheme:
+
+    ![Create a new folder](spring-boot-sql-scheme.png)
+
+2. Update the `src/main/resources/sql/schema.sql` file with the following code:
+
+    ```sql
+    CREATE TABLE IF NOT EXISTS messages (
+      id                     VARCHAR(60)  DEFAULT RANDOM_UUID() PRIMARY KEY,
+      text                   VARCHAR      NOT NULL
+    );
+    ```
+
+   It creates the `messages` table with two fields inside: `id` and `text`. The table structure is the same as `Message` class.
+   
+3. Open the `application.properties` located in the `src/main/resources` folder, and add the following application properties:
+
+    ```properties
+    spring.datasource.driver-class-name=org.h2.Driver
+    spring.datasource.url=jdbc:h2:file:./data/testdb
+    spring.datasource.username=sa
+    spring.datasource.password=password
+    spring.datasource.schema=classpath:sql/schema.sql
+    spring.datasource.initialization-mode=always
+    ```
+
+    These settings enable the database to the Spring Boot application.
+    See the full list of common application properties in the [Spring documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html).
+
+## Run the application
+
+You should use an HTTP client to work with previously created endpoints.
+In IntelliJ IDEA, you can do that by using the embedded [HTTP client](https://www.jetbrains.com/help/idea/http-client-in-product-code-editor.html): 
+
+1. [Run the application](#run-the-application). Once the application is up and running, you can execute POST requests to store messages in the database.
+
+1. Create the `requests.http` file and add the following HTTP requests:
+
+    ```http request
+    ### Post 'Hello!"
+    POST http://localhost:8080/
+    Content-Type: application/json
+    
+    {
+      "text": "Hello!"
+    }
+    
+    ### Post "Bonjour!"
+    
+    POST http://localhost:8080/
+    Content-Type: application/json
+    
+    {
+      "text": "Bonjour!"
+    }
+    
+    ### Post "Privet!"
+    
+    POST http://localhost:8080/
+    Content-Type: application/json
+    
+    {
+      "text": "Privet!"
+    }
+    
+    ### Get all the messages
+    GET http://localhost:8080/
+    ```
+
+2. Execute all POST requests. Use the green **Run** icon in the gutter to the request declaration.
+   These requests write the text messages to the database.
+    
+    ![Run HTTP POST requests](spring-boot-run-http-request.png)
+
+3. Execute the GET request and see the result in the **Run** tool window:
+
+    ![Run HTTP GET request](spring-boot-output-2.png)
+
+> You can also use any other HTTP client or cURL command-line tool. For example, you can run the following commands in the terminal to get the same result:
```

Comment:
Maybe make this a separate section? Seems like a complete topic - an alternative way to execute requests. Too broad for a note.